src/gobupload/update.py

```diff
@@ -218,8 +216,7 @@ def full_update(msg):
     :param msg: the result of the application of the events
     :return: Result message
     """
-    global logger
-    logger = init_logger(msg, "UPDATE")
+    logger.configure(msg, "UPDATE")
```

Comment:
Uses of the logger instance within this file can only happen after logger.configure.
Any uses of logger before, will result in exception since self._name is not set.

Reply:
Good point, throw an assertion on missing config

Reply:
In the logger class of course

Reply:
What are your other remarks on the solution?